browser_patches/webkit/archive.sh

```diff
@@ -75,11 +75,15 @@ createZipForLinux() {
     # Copy libvpx.so.5 as Ubuntu 20.04 comes with libvpx.so.6
     ldd WebKitBuild/WPE/Release/bin/MiniBrowser | grep -o '[^ ]*\/libvpx.so.5[^ ]*' | xargs cp -t $tmpdir
     # Copy some wayland libraries required for Web Process t
-    cp -d -t $tmpdir WebKitBuild/WPE/DependenciesWPE/Root/lib/libva\-*
+    if ls WebKitBuild/WPE/DependenciesWPE/Root/lib/libva\-* 2>&1 >/dev/null; then
+      cp -d -t $tmpdir WebKitBuild/WPE/DependenciesWPE/Root/lib/libva\-*
+    fi
     # Injected bundle is loaded dynamicly via dlopen => not bt listed by ldd.
     cp -t $tmpdir WebKitBuild/WPE/Release/lib/libWPEInjectedBundle.so
-    mkdir -p $tmpdir/gio/modules
-    cp -t $tmpdir/gio/modules $PWD/WebKitBuild/WPE/DependenciesWPE/Root/lib/gio/modules/*
+    if test -d $PWD/WebKitBuild/WPE/DependenciesWPE/Root/lib/gio/modules/; then
```

Comment:
I assume that this if-condition means that there are no gio modules to copy if we build with Ubuntu-18.

However, I remember not having these were breaking our "ignore https errors" tests on Github Actions.

Anyhow, we can land this and then see how it works on our CI!

Reply:
That's right, with the minimal jhbuild moduleset no extra gio modules are built, only the system ones are used. The script however is adapted to check for any gio module built and copy it (if its there)